fn_risk_fabric/setup.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name='fn_risk_fabric',
+    version='1.0.0',
+    license='Resilient License',
```

Comment:
should correspond to the 'LICENSE' file (which is MIT in the current PR, but feel free to use whatever you choose)